src/environment.h

```diff
@@ -379,6 +378,11 @@ class ServerEnvironment : public Environment
 	// is weather active in this environment?
 	bool m_use_weather;
 	ABMHandler * m_abmhandler;
```

Comment:
make this a std::unique_ptr so it's automatically deleted in destructor, that's if it really has to be created dynamically with `new`, is there any reason for that?